packages/webhooks/lib/sync.ts

```diff
@@ -80,7 +80,7 @@ export const sendSync = async ({
                 deleted: 0
             },
             modifiedAfter: dayjs(now).toDate().toISOString(),
-            queryTimeStamp: operation !== 'INITIAL' ? (now as unknown as string) : null
+            queryTimeStamp: now as unknown as string // Deprecated
```

Comment:
does this need to change? Can you go through why this is needed?

Reply:
no sync is ever set as `INITIAL` anymore so this condition is always true and therefore unnecessary